### DISCUSSION TITLE
Removal of .NET 5 runtime from all SDK images

### DISCUSSION THREAD

**Discussion by mthalman** [microsoft/dotnet-framework-docker#1006] [Announcements]
# Removal of .NET 5 runtime from all SDK images

## Summary

Effective in today's servicing release (September 2022), the .NET 5 runtime has been removed from all SDK images because it reached its end-of-life date on May 10, 2022.

## Details

* https://github.com/microsoft/dotnet-framework-docker/pull/993

This affects all supported [SDK tags documented on Docker Hub](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/).

**Comment by eriawan** [comment]
@mthalman 

Do you mean the link in this:
> This affects all supported [SDK tags documented on Docker Hub](https://hub.docker.com/_/microsoft-dotnet-framework-sdk/).

That link is supposed to be .NET, not .NET Framework right? 
I mean it's supposed to be this: https://hub.docker.com/_/microsoft-dotnet/ or this link: https://hub.docker.com/_/microsoft-dotnet-sdk/

**Reply by mthalman** [reply]
No, this applies to the .NET Framework SDK container images. As seen in the associated PR (https://github.com/microsoft/dotnet-framework-docker/pull/993), the .NET 5 runtime was originally installed via the VS Build Tools installation in the .NET Framework SDK. This has now been removed.

**Comment by mthalman** [comment]
I've updated this announcement. It originally referred to the .NET 5 SDK being removed. I've corrected it to refer to the .NET 5 _runtime_ instead.
